osbenchmark/utils/git.py

```diff
@@ -119,7 +119,7 @@ def current_branch(src_dir):
 def branches(src_dir, remote=True):
     clean_src = io.escape_path(src_dir)
     if remote:
-        # alternatively: git for-each-ref refs/remotes/ --format='%(refname:short)'
+        # Because compatability issues with Git 2.40.0+, updated --format='%(refname:short)' to --format='%(refname)'
         return _cleanup_remote_branch_names(process.run_subprocess_with_output(
```

Comment:
Do we have any existing test for this that need to be updated.

Reply:
Yes, it was updated in this PR https://github.com/opensearch-project/opensearch-benchmark/pull/246/files#diff-b9982f3646bdc2d39546ab7c9316d4bdeaf1bd49ad88f9dd27dff64494e0ea63R182